client/src/components/Projects/index.jsx

```diff
@@ -1,12 +1,14 @@
-import React from "react";
+import React, { Component } from "react";
 import { Link } from "react-router-dom";
 
-export default function Projects() {
-  return (
-    <div>
-      Projects
-      <Link to={'/project/1/1'}>project 1</Link>
-      <Link to={'/project/2/1'}>project 2</Link>
-    </div>
-  );
+export default class Projects extends Component {
+  render() {
+    return (
+      <div>
+        Projects
+        <Link to={"/project/1/1"}>project 1</Link>
+        <Link to={"/project/2/1"}>project 2</Link>
+      </div>
+    );
+  }
```

Comment:
FGS
why did you commit these changes :sob:

Reply:
remove change